src/rust/ide/src/model/synchronized/graph.rs

```diff
@@ -0,0 +1,3 @@
+//! Synchronized graph models.
+
+pub mod executed;
```

Comment:
I really dont like this mod name. We don't have names in a form of past verbs here. Could we somehow improve it?

Reply:
How would you name a module containing `ExecutedGraph` structure inside `graph` module?